tests/console/upload_asset.pm

```diff
@@ -18,9 +18,7 @@ use scheduler "get_test_suite_data";
 
 sub run {
     select_console 'root-console';
-    my $files_to_upload = get_test_suite_data()->{asset_files};
-    assert_script_run("tar -cvf asset_files.tar $files_to_upload", fail_message => "Failed to compress file(s)");
```

Comment:
It is visible that the module "upload_asset" was meant to be generic, hence the usage of test data. However in practice, this module is used in only one place, and is redundant as it collects again the files that were already collected in the previous module "logs_from_installation_system". 
This change uses the "strict expectation" approach, we could also keep using the "no test data in modules" approach: just change test data, and only change this line to upload existing logs instead of creating a new archive.